src/routes/reset-password/reset-request.ts

```diff
@@ -0,0 +1,39 @@
+import { FastifyInstance, FastifyReply, FastifyRequest } from 'fastify';
+import {
+  passwordResetRequestBody,
+  PasswordResetRequestBody,
+  passwordResetResponseBody,
+} from '../../types/auth.types';
+import { handlePasswordResetRequest } from '../../lib/queries/resetRequest';
+
+const schema = {
+  tags: ['Auth'],
+  description: 'Responds with 202 Accepted regardless of unverified email.',
+  body: passwordResetRequestBody,
+  response: {
+    202: passwordResetResponseBody,
+    400: { $ref: 'error' },
+  },
+};
+
+schema.body.properties.email.example = 'john.doe@example.com';
+
+const handler = async (
+  request: FastifyRequest<{ Body: PasswordResetRequestBody }>,
+  reply: FastifyReply
+): Promise<void> => {
+  await reply
+    .code(202)
+    .send({ message: 'Password reset email is being sent.' });
+
+  return handlePasswordResetRequest(request.body);
```

Comment:
```suggestion
  await handlePasswordResetRequest(request.body);
```

Funktion paluuarvo on myös reply, joskaan sitä ei lähetetä mutta fastify valittaa (tai ainakin valitti ennen) logeissaan kahdesta vastauksesta clientille. Nyt kun reply laitetaan jo alussa niin ei palauteta mitään funktion lopussa. Await saattaa olla kuitenkin paikallaan, että mahdolliset errorit napataan kiinni

Reply:
Ei vaan se palauttaa tyhjän Promisen

Reply:
sekä tuo handleri, että tuo `handlePasswordResetRequest`